Basic-Car-Maintenance/Shared/Settings/Views/ContributorsListView.swift

```diff
@@ -9,7 +9,11 @@ import SwiftUI
 
 struct ContributorsListView: View {
     
-    @ObservedObject var viewModel: SettingsViewModel
+    var viewModel: SettingsViewModel
+
+    init(viewModel: SettingsViewModel) {
```

Comment:
Removing this `init` cause it's not needed, structs create their own `init`s and there's nothing special about this one